fe/fe-core/src/main/java/com/starrocks/catalog/SchemaTable.java

```diff
@@ -589,6 +609,8 @@ public enum SchemaTableType {
         SCH_TASK_RUNS("TASK_RUNS", "TASK_RUNS", TSchemaTableType.SCH_TASK_RUNS),
         SCH_VERBOSE_SESSION_VARIABLES("VERBOSE_SESSION_VARIABLES", "VERBOSE_SESSION_VARIABLES",
                 TSchemaTableType.SCH_VERBOSE_SESSION_VARIABLES),
+        SCH_BE_TABLETS("BE_TABLETS", "BE_TABLETS",
```

Comment:
+1
I'd like the name of "tablets". Because 
1. later we may change the BE name. 
2. when SR become separation of computing and storage, tablets may have no relations with BE.

Reply:
be_tablets, it's different from tablets/replicas in FE,  also I plan to add more BE metrics, like memory, diskio, tasks, logs etc all with be_ prefix.

Reply:
> +1 I'd like the name of "tablets". Because
> 
> 1. later we may change the BE name.
> 2. when SR become separation of computing and storage, tablets may have no relations with BE.

In that case, this PR wont work, we need compute node and tablets in the same node to execute related scanner.